src/platform_impl/ios/monitor.rs

```diff
@@ -35,15 +38,20 @@ impl Drop for VideoMode {
     fn drop(&mut self) {
         unsafe {
             assert_main_thread!("`VideoMode` can only be dropped on the main thread on iOS");
-            msg_send![self.screen_mode, release];
+            msg_send![self.screen_mode, release]
```

Comment:
Piggybacking off of `drop`'s return type for this is very cute, but I'm worried it's a bit too opaque - it's a case where re-ordering the lines would change the semantics in surprising ways.